### ISSUE TITLE
How to generate coco_glove_word2vec.pkl? Could you share your code?Thank you!

### ISSUE THREAD

**Issue by yfreedomliTHU** [chenzhaomin123/ML_GCN#10]

**Comment by monk1337** [comment]
You have labels, use word2vec to get multidimensional vector for each label.
You can check this repo for more info : 

https://github.com/monk1337/word_embedding-in-tensorflow

**Comment by 2014gaokao** [comment]
word_to_ix = {"person": 0, "car": 1,...}
embeds = nn.Embedding(20, 300) 

emmm, is this enough?
